acceptance/setup/git/pre-suite/000_EnvSetup.rb

```diff
@@ -112,7 +112,8 @@
 
     step "#{host} Install ruby from git using revision #{revision}"
     # TODO remove this step once we are installing puppet from msi packages
```

Comment:
comment could be removed since this is in the git setup suite.  but 😐 it shouldn't hold up this PR